### ISSUE TITLE
Updating documentation for react-plugin

### ISSUE THREAD

**Issue by eloudsa** [ConsenSys-archive/drizzle#53]
The documentation describing how to install and to use react-plugin is still related to the legacy version:
https://github.com/trufflesuite/drizzle/blob/develop/packages/react-plugin/README.md

Any plan to update it for Drizzle 1.5?

**Comment by cds-amal** [comment]
Hi @eloudsa,

The documentation should be updated! I have a few items on my plate and marked this as a `good first issue` to encourage contributors. PRs welcome :)

**Comment by cayblood** [comment]
Gotta admit that it is pretty difficult to get started with so much inconsistency between the box demo, the tutorials, the docs and the READMEs. Moving from the box demo to custom components is not very intuitive.

**Comment by adrianmcli** [comment]
@cayblood 100% agreed. We are currently struggling to divide our capacity to keep these things up-to-date. We're working on it, but we know we still have a long way to go. This is why we definitely welcome any PRs if the community wants to help out!

**Comment by cayblood** [comment]
Thanks @adrianmcli. The challenge is that you have a chicken-and-egg problem. Many of us are not sufficiently proficient in React to decipher the current code base and update all the relevant docs, and the code base is itself in a state of flux. My recommendation to you guys is to not accept any PRs from any contributor that don't also update ALL relevant docs. Each commit should maintain documentation consistency. This doesn't add a lot of overhead if done regularly, but it becomes harder and harder to catch up if it's not maintained. This also becomes easier if all your tutorials and docs are generated from the same repo, so that updates stay atomic.

**Comment by adrianmcli** [comment]
Great point @cayblood, we'll discussion your recommendations in our standup. Just to recap your points:

- make sure PRs always include updated docs
- make sure all tutorials and docs are generated from the same repo

attn: @cds-amal

**Comment by NTP-996** [comment]
Is this issue still available? i want to work on this

**Comment by adrianmcli** [comment]
@NTTMai yes! we have shifting priorities at Truffle and sometimes we don't have enough bandwidth to take care of all our public repos, so any help from the community is always welcome!

**Comment by ghost** [comment]
Hi there, I'd like to work on this but i'm not sure if @NTTMai is in progress. 

If so, would it be appropriate to replace the documentation for the legacy api with a link to ~~the legacy repo's readme~~ a LEGACYAPI.md? Thanks.
